project.clj

```diff
@@ -30,7 +30,6 @@
             [lein-pprint "1.1.1"]
             [lein-ring "0.9.7"]]
   :main datomic-rest-api.get-handler
```

Comment:
@mgrbyte this makes sense to me. Our configuration was consistently working with the aot. It was just the runtime that was having troubles.